### ISSUE TITLE
Git status files are not recognised when in root directory

### ISSUE THREAD

**Issue by mirfilip** [facebook/PathPicker#284]
When you are in a git repo and you deal with a file in a root directory of a repo, you can't choose it with `fpp`:
```
$ file .git
.git: directory // I'm in the repo
$ mkdir test123 && touch test123/asdf
$ touch test
$ git status
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	test
	test123/
$ git status | fpp
No lines matched!!
```

When you git add `test123/asdf`, `fpp` can then properly parse it as a path.

I presume this can be similar to #20 and is a consequence of the format of `git status` (it just displays a filename, not `./filename`).

**Comment by mirfilip** [comment]
`status.relativePaths` config for `git status` does not help, as it doesn't change the way status is displayed when you don't move into subdirs.

**Comment by pcottle** [comment]
yep agreed that it doesn't handle all cases, and theres zero support for directories as well. in hindsight I would have:
 - supported directories out of the box (and even had different default commands for them perhaps)
 - perhaps been more opinionated about the type of input I received so I could match files with no extension like `test`, or had specialized git status input modes where I bypass the generalized parsing rules.

either way despite working at FB, its not like I get time allocated to make improvements to fpp :P any support I do is at the expense of my actual professional responsibilities, and simply because I want to make sure the project gets a modicum of support

**Comment by mirfilip** [comment]
Sure, I understand. Thx for the tool in the first place! My only moan would be to include this limitation in the docs, this is a 3800 stars project after all.

**Comment by pcottle** [comment]
Good point, updated the readme
